src/main/java/com/google/sps/servlets/SentimentServlet.java

```diff
@@ -103,4 +137,15 @@ private Float calculateSentimentScore(String text) throws ApiException {
 
     return score;
   }
+
+  /**
+   * Creates a Json String of a VideoAnalysis object.
+   */ 
+  private String createResponseJson(Float score, boolean available) {
```

Comment:
I suggest trying to use primitive types (e.g. float instead of Float) wherever possible because they are not nullable.